src/test/resources/full_marc.jsonl

```diff
@@ -102,6 +102,15 @@
         ]
       }
     },
+    {
+      "043": {
+        "subfields": [
+          {
+            "k": "asd"
+          }
+        ]
+      }
+    },
```

Comment:
minor
I suggest adding separate test cases to improve our test base and general quality.

Reply:
done